organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
+open society.
+
+**Open Society**: An open society promotes freedom of thought and expression,
+personal responsibility, individualism, cooperation, and equality of all
+citizens.
+
+Code of Conduct
+===============
+
+It is expected that members behave in a way that allows Hacker Society to
+achieve its mission.  Participants in Hacker Society events are expected to
+treat each other with dignity and respect for each other's perspectives,
+experiences, and knowledge.  In particular, participants should recognize that
+people of all skill levels are welcome, and should always be willing to help
+others learn.
+
+To support Hacker Society's mission of: learning technology from each other,
+building community, and promoting equality, Hacker Society must be a place where
+people can be comfortable regardless of gender, gender identity and expression,
+sexual orientation, disability, physical appearance, body size, race, age, or
+religion. Therefore, we do not tolerate:
+
+- Rude or unwelcome comments about a person's gender, gender identity, gender
+  expression, sexual orientation, disability, physical appearance, attire, race,
+  age or religious expression.
+- Sexual images in public spaces
+- Deliberate intimidation, stalking, or following
+- Harassing photography or recording
+- Sustained disruption of talks or other events
+- Inappropriate physical contact
+- Unwelcome sexual attention
+- Advocating for, or encouraging, any of the above behavior
+
+Necessarily, the goals of learning, community, and equality are in tension
```

Comment:
> Although on the other hand, significant technical (or other) contributions should not excuse unacceptable behaviour.

@matthewbentley: Totally agreed.  Two things to add to what @timtadh says about this:

1. In the cases Tim's talking about, I think people's behavior depends largely on who's receiving it.  If two people have a certain rapport where they are rude to each other in jest, and someone's comments are received that way, then I don't see a problem.  So long as others aren't being dragged in by wide-ranging offensive comments.
2. In my opinion, no amount of technical contribution excuses somebody from inappropriate, harassing behavior.  Someone who contributes technically but can't behave appropriately is clearly capable of behaving, but chooses not to.  They need to learn to behave appropriately before they can take part in the community.  I think this tradeoff is mostly important when people are discussing sensitive topics.  For example, discussions about gender, sexuality, etc., may be uncomfortable for some people, but so long as the discussion is being conducted in a respectful manner, they should be allowed.